components/shared/identity/IdentityChain.vue

```diff
@@ -0,0 +1,60 @@
+<template v-else>
+  <div>
+    <span
+      v-if="showOnchainIdentity"
+      class="is-inline-flex is-align-items-center">
+      {{ shortenedAddress }}
+      <img
+        v-if="isFetchingIdentity"
+        src="/infinity.svg"
+        class="ml-1 infinity-loader" />
+      <template v-else>
+        <span v-if="identity.display" class="ml-1"
+          >({{ identity.display }})</span
+        >
+      </template>
+    </span>
+    <template v-else-if="!hideIdentityPopover">
+      <IdentityPopover :identity="{ ...identity, address }">
+        <template #trigger>
+          {{ name }}
+        </template>
+      </IdentityPopover>
+    </template>
+    <span v-else>
+      {{ name }}
+    </span>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { GenericAccountId } from '@polkadot/types/generic/AccountId'
+
+import { emptyObject } from '@/utils/empty'
+
+type IdentityFields = Record<string, string>
+type Address = string | GenericAccountId | undefined
+
+const components = {
+  IdentityPopover: () =>
+    import('@/components/shared/identity/popover/IdentityPopover.vue'),
+}
+
+@Component({ components })
+export default class IdentityChain extends Vue {
+  @Prop() readonly showOnchainIdentity!: boolean
+  @Prop() readonly hideIdentityPopover!: boolean
+  @Prop() readonly isFetchingIdentity!: boolean
+  @Prop({ default: emptyObject<IdentityFields>() }) readonly identity
+  @Prop() readonly address!: Address
+  @Prop() readonly shortenedAddress!: Address
+  @Prop() readonly name!: Address
+}
+</script>
+
+<style scoped>
+.infinity-loader {
+  height: 20px;
+}
+</style>
```

Comment:
is it needed?

Reply:
needed when loading, tried it with `height="20"` not working in image tag. any suggestion?

Reply:
It seems strange to me why this rule is necessary knowing that there is no `infinity-loader` in component, is it displaying when loaded?

Reply:
oh, the line used in the line of 10 on IdentityChain.vue. I forgot to move into that component before in the previous commit 🙏🏻